main/main.cpp

```diff
@@ -792,8 +752,6 @@ Error Main::test_setup() {
 	// Theme needs modules to be initialized so that sub-resources can be loaded.
 	theme_db->initialize_theme_noproject();
 
```

Comment:
```suggestion

		NavigationServer3DManager::initialize_server(); // 3D server first because 2D depends on it.
		NavigationServer2DManager::initialize_server();

```